src/main/kotlin/com/team4099/robot2023/subsystems/manipulator/Manipulator.kt

```diff
@@ -0,0 +1,92 @@
+package com.team4099.robot2023.subsystems.manipulator
+
+import com.team4099.lib.hal.Clock
+import com.team4099.lib.logging.LoggedTunableValue
+import com.team4099.robot2023.config.constants.ManipulatorConstants
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import org.littletonrobotics.junction.Logger
+import org.team4099.lib.controller.SimpleMotorFeedforward
+import org.team4099.lib.units.base.inSeconds
+import org.team4099.lib.units.derived.inVoltsPerMeter
+import org.team4099.lib.units.derived.inVoltsPerMeterPerSecond
+import org.team4099.lib.units.derived.inVoltsPerMeterSeconds
+import org.team4099.lib.units.derived.perMeter
+import org.team4099.lib.units.derived.perMeterPerSecond
+import org.team4099.lib.units.derived.perMeterSeconds
+import org.team4099.lib.units.derived.volts
+
+class Manipulator(val io: ManipulatorIO) : SubsystemBase() {
```

Comment:
Is there a way for this subsystem to be aware of which game piece it has? (hint: there is). I think it is harder to know if it currently is holding something, but that should be doable too.